server/synchronization/task_executor.go

```diff
@@ -197,6 +198,44 @@ func (te *TaskExecutor) execute(i interface{}) {
 		te.handleError(task, taskLogger, msg, true)
 		return
 	}
+	te.onSuccess(task, taskLogger)
+}
+
+func (te *TaskExecutor) onSuccess(task *meta.Task, taskLogger *TaskLogger) {
+	var dbtClouds = te.destinationService.GetDestinationsByType(storages.DbtCloudType)
```

Comment:
We should filter by project ID as well. Usually, it happened in the Configurator backend, where we just set destination IDs into source configuration.

Reply:
Ouch. Thanks